bats/fb-katello-content.bats

```diff
@@ -116,6 +106,145 @@ setup() {
     --content-view="${CONTENT_VIEW}" --to-lifecycle-environment="${LIFECYCLE_ENVIRONMENT}" --from-lifecycle-environment="Library"
 }
 
+@test "export content view version" {
```

Comment:
I wonder if this should be a separate file. To me it feels like a different test than the regular content view tests.

Reply:
@ekohl Separate files would make sense if the setup on these import/export tests did not rely on artifacts created before.

@jeremylenz  I suggest adding method to check if its at least Katello 3.18. We can skip each of these import/export tests if its not katello 3.18 at least. @ekohl would that help with your concern ?

Reply:
Yes, I think it would.

Reply:
@ekohl Added `tSkipIfOlderThan318`